universal-application-tool-0.0.1/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -78,7 +78,7 @@ private Tag renderDownloadButton(long programId, long applicationId) {
         .setText("Download (PDF)")
         .asButton()
         // TODO: when the download link works, un-hide.
-        .isHidden();
+        .withClasses(Styles.HIDDEN);
```

Comment:
(blocking) according to https://github.com/seattle-uat/civiform/pull/1295/files, `ProgramApplicationListView` line 173 also needs this change.

Reply:
oh nvm anchors have `hidden`, but not buttons.